### DISCUSSION TITLE
Websearch with company proxy server not working

### DISCUSSION THREAD

**Discussion by codingstube** [open-webui/open-webui#7332] [Issues]
Hi everyone, 

we are running open-webui behind a company http proxy to reach internetsites. With the newer versions of open-webui > 0.4 the websearch does not work.

 Fetching pages:   0%|          | 0/3 [00:00<?, ?it/s]ERROR [open_webui.apps.openai.main] Connection error:
WARNI [langchain_community.document_loaders.web_base] Error fetching https://www.auto-hus.eu/ with attempt 1/3: Cannot connect to host www.auto-hus.eu:443 ssl:default [Connect call failed ('34.149.87.45', 443)]. Retrying...

the http and https_proxy is in the environment of the docker container but seems to be ignrored. 
Furthermore it makes no different if I choose duckduckgo as websearch or our searxng server. The searchxng server works. 

Does anybody know how to configure open-webui in the newer version to use the company proxy server?

Thanks

**Comment by codingstube** [comment]
Ok I see that searxng actually search for sites, but the answer did not get to open-webui.

**Reply by Joni1717** [reply]
Hello codingstube, we have exactly the same issue. Have you possibly found a fix for this you can share?
